dtfs-central-api/src/utils/report-period.ts

```diff
@@ -4,10 +4,7 @@ import { IsoMonthStamp } from '../types/date';
 import { ReportPeriodStart } from '../types/utilisation-reports';
 import { UtilisationReport } from '../types/db-models/utilisation-reports';
 
-export const getReportPeriodStartForUtilisationReport = ({ month, year }: UtilisationReport): ReportPeriodStart => ({
-  month,
-  year,
-});
+export const getReportPeriodStartForUtilisationReport = ({ reportPeriod }: UtilisationReport): ReportPeriodStart => (reportPeriod.start);
```

Comment:
I think we can probably remove this function now as `reportPeriod.start` is now more concise that calling the function : )